### PR TITLE
Fixed issues with dependencies

### PR DESCRIPTION
**Pull request by nakytoe**
## Description

The template had unused dependency for pysqlite3, however python's internal sqlite3 is used instead. Dependency removed .
The nb-black dependency is outdated and can not be installed breaking new containers. An issue has been raised and a solution provided, but until the package is updated, the dependency is deprecated.

### Manual testing
Manually tested ml pipeline, model store,  API and logging after the changes.

### CODE REVIEW

File: requirements/dev-requirements.in [hunk 1]
```diff
@@ -2,7 +2,8 @@
 nbdev
 pre-commit
 black
-nb-black
+# nb-black (outdated install requirements, can not be installed)
```
Comment (jussiarpalahti) [inline requirements/dev-requirements.in:5]: Black comes with Jupyter support these days:
`black`
can be changed to
`black[jupyter]`

https://black.readthedocs.io/en/stable/getting_started.html#installation

Comment (nakytoe) [reply requirements/dev-requirements.in]: changed to black, removed nb-black completely

**Comment by nakytoe** [comment]
- switched from nb-black to black[jupyter]. Not as convenient because notebook cleanup is performed as recommit action, but it works.
- solved distutils dependency issue by uninstalling blinker in docker file before requirements are installed

**Review by jussiarpalahti — APPROVED** [review]
LGTM

**Pull request merged**